Library/Formula/icecast.rb

```diff
@@ -1,5 +1,3 @@
-require 'formula'
-
```

Comment:
Whoops - so used to doing that now that it's a matter of instinct.  I'll take that out.